src/containers/calculations.js

```diff
@@ -113,7 +159,19 @@ class CalculationsContainer extends Component {
     if (this.props.match.params.id) {
       creatorId = this.props.match.params.id;
     }
+
+    const {sortIndex} = this.state;
+    const {sortdir, sort, limit, offset} = pagination;
+    for (let val in search) {
+      if (search[val] === undefined) {
```

Comment:
Let use lodash `isUndefined` for this test.

Reply:
Fixed